tests/schema/tests.py

```diff
@@ -3504,6 +3504,47 @@ def test_unique_constraint_nulls_distinct_unsupported(self):
             self.assertIsNone(editor.add_constraint(Author, constraint))
             self.assertIsNone(editor.remove_constraint(Author, constraint))
 
+    @skipUnlessDBFeature("supports_nulls_distinct_unique_constraints")
+    def test_create_unique_nulls_distinct(self):
```

Comment:
Maybe this test should be renamed to mention the fact that this will result in a constraint and the other test in an index as it relies on expressions?